diff --git a/src/EasyBreadcrumbBuilder.php b/src/EasyBreadcrumbBuilder.php
index 9a6c2f6..15834bf 100644
--- a/src/EasyBreadcrumbBuilder.php
+++ b/src/EasyBreadcrumbBuilder.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\easy_breadcrumb\EasyBreadcrumbBuilder.
- */
-
 namespace Drupal\easy_breadcrumb;
 
 use Drupal\easy_breadcrumb\EasyBreadcrumbConstants;
diff --git a/src/EventSubscriber/InitSubscriber.php b/src/EventSubscriber/InitSubscriber.php
index 1ab8c4d..8fec515 100644
--- a/src/EventSubscriber/InitSubscriber.php
+++ b/src/EventSubscriber/InitSubscriber.php
@@ -1,8 +1,4 @@
 <?php
-/**
- * @file
- * Contains \Drupal\easy_breadcrumb\EventSubscriber\InitSubscriber.
- */
 
 namespace Drupal\easy_breadcrumb\EventSubscriber;
 
diff --git a/src/Form/EasyBreadcrumbGeneralSettingsForm.php b/src/Form/EasyBreadcrumbGeneralSettingsForm.php
index c96423a..295721a 100644
--- a/src/Form/EasyBreadcrumbGeneralSettingsForm.php
+++ b/src/Form/EasyBreadcrumbGeneralSettingsForm.php
@@ -1,10 +1,5 @@
 <?php
 
-/**
- * @file
- * Contains \Drupal\easy_breadcrumb\Form\EasyBreadcrumbGeneralSettingsForm.
- */
-
 namespace Drupal\easy_breadcrumb\Form;
 
 use Drupal\Core\Form\ConfigFormBase;
