diff --git a/src/EasyBreadcrumbBuilder.php b/src/EasyBreadcrumbBuilder.php
index 9da15bd..9a3b3fb 100644
--- a/src/EasyBreadcrumbBuilder.php
+++ b/src/EasyBreadcrumbBuilder.php
@@ -169,6 +169,7 @@ class EasyBreadcrumbBuilder implements BreadcrumbBuilderInterface {
     $breadcrumb->addCacheContexts(['url.path']);
     $breadcrumb->addCacheableDependency($this->config);
     $i = 0;
+    $add_langcode = FALSE;
 
     // Remove the current page if it's not wanted.
     if (!$this->config->get(EasyBreadcrumbConstants::INCLUDE_TITLE_SEGMENT)) {
@@ -181,13 +182,18 @@ class EasyBreadcrumbBuilder implements BreadcrumbBuilderInterface {
       if (!($this->config->get(EasyBreadcrumbConstants::LANGUAGE_PATH_PREFIX_AS_SEGMENT))) {
         if (Unicode::strtolower($path_elements[0]) == $curr_lang) {
           array_shift($path_elements);
+          $add_langcode = TRUE;
         }
       }
     }
     while (count($path_elements) > 0) {
+      $check_path = '/' . implode('/', $path_elements);
+      if ($add_langcode) {
+        $check_path = '/' . $curr_lang . $check_path;
+      }
 
       // Copy the path elements for up-casting.
-      $route_request = $this->getRequestForPath('/' . implode('/', $path_elements), $exclude);
+      $route_request = $this->getRequestForPath($check_path, $exclude);
       if ($this->config->get(EasyBreadcrumbConstants::EXCLUDED_PATHS)) {
         $config_textarea = $this->config->get(EasyBreadcrumbConstants::EXCLUDED_PATHS);
         $excludes = preg_split('/[\r\n]+/', $config_textarea, -1, PREG_SPLIT_NO_EMPTY);
